### PR TITLE
Fix disclaimer

### PR DESCRIPTION
**Pull request by FlorianJacta**

### CODE REVIEW

File: docs/gallery/decision_support/3_production_planning/index.md [hunk 1]
```diff
@@ -112,7 +112,3 @@ Access and display various tables associated with
 a selected scenario. Conveniently download data tables in CSV format.
 
 ![Databases](images/production-planning-databases.png){width=90% : .tp-image-border }
-
-# Disclaimer: Enterprise License
-
-The code for this demo is only available in the Taipy Enterprise License.
```
Comment (jrobinAV) [inline docs/gallery/decision_support/3_production_planning/index.md:118]: Why did you remove the disclaimer?

**Review by jrobinAV — APPROVED** [review]

File: docs/gallery/decision_support/3_production_planning/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/gallery/decision_support/3_production_planning/index.md:118]: It is not on this demo
